pkgs/top-level/all-packages.nix

```diff
@@ -15945,6 +15947,15 @@ in
 
   blas = callPackage ../development/libraries/science/math/blas { };
 
+  clblas-cuda = callPackage ../development/libraries/science/math/clblas/cuda {
+    cudatoolkit = pkgs.cudatoolkit75;
+    inherit (linuxPackages) nvidia_x11;
+    python = pkgs.python34Packages.python;
```

Comment:
Is there a specific reason you chose Python 3.4? Is this to be used also outside of Python environments?